cmds/core/netcat/netcat.go

```diff
@@ -255,15 +273,15 @@ func evalParams(args []string, f flags) (*netcat.Config, error) {
 
 	config.AccessControl, err = netcat.ParseAccessControl(f.connectionAllowFile, connectionAllowList, f.connectionDenyFile, connectionDenyList)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("%w: %v", os.ErrInvalid, err)
```

Comment:
missed that in my prior review, same thing here breaking the CI
```suggestion
		return nil, fmt.Errorf("%w: %w", os.ErrInvalid, err)
```